### PR TITLE
Update/redirect links to website

### PR DESCRIPTION
**Pull request by sambhav2612**
Hey, I'm interested in maintaining this repo. Please have a look at the changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -214,30 +214,29 @@ These are provided as examples of C game programming code.
...
+* [libao](https://xiph.org/ao/) - Cross-platform audio library with a wide variety of outputs. [`GNU GPL2.1 or later`](http://www.gnu.org/licenses/old-licenses/gpl-2.0.html)
+* [RetroArch](https://www.libretro.com/) - The reference frontend for [libretro](https://www.libretro.com/). [`GNU GPL3`](http://www.gnu.org/licenses/gpl.html)
+* [SDL and SDL2](https://www.libsdl.org/) - Cross-platform development library designed to provide low-level access to audio, keyboard, mouse, joystick and graphics hardware via OpenGL. SDL2 is the most current version. [`zlib`](https://directory.fsf.org/wiki/License:Zlib)
+* [raylib](http://www.raylib.com/) - Simple and easy-to-use library to learn videogames programming. [zlib](https://directory.fsf.org/wiki/License:Zlib)
 
 ## Generic Programming ##
 
-* [klib](https://github.com/attractivechaos/klib) - Small and lightweight implementations of common algorithms and data structures. [`Expat`](https://directory.fsf.org/wiki/License:Expat)
+* [klib](http://attractivechaos.github.io/klib/#About) - Small and lightweight implementations of common algorithms and data structures. [`MIT`](https://directory.fsf.org/wiki/License:MIT)
```
Comment (sjfricke) [inline README.md]: There is not `MIT` in the FSF wiki, Travis CI gets a 404

File: README.md [hunk 2]
```diff
@@ -175,10 +175,10 @@ These are projects that allow one to work with microcontrollers and other resour
...
 * [ChibiOS](http://www.chibios.org/dokuwiki/doku.php) - RTOS with complete development environment (HAL, drivers, support files, and tools). [`Various Licenses`](http://www.chibios.org/dokuwiki/doku.php?id=chibios:licensing:start)
-* [Contiki](https://github.com/contiki-os/contiki) - Connect low-cost, low power microcontrollers to the Internet. [`3-clause BSD`](https://github.com/contiki-os/contiki/blob/master/LICENSE)
-* [Zephyr Project](https://github.com/zephyrproject-rtos/zephyr) - Scalable real-time RTOS supporting multiple hardware architectures, optimized for resource constrained devices. [`Apache 2.0`](https://directory.fsf.org/wiki/License:Apache2.0)
+* [Contiki](http://www.contiki-os.org/) - Connect low-cost, low power microcontrollers to the Internet. [`3-clause BSD`](https://github.com/contiki-os/contiki/blob/master/LICENSE)
+* [Zephyr Project](http://zephyrproject.org/) - Scalable real-time RTOS supporting multiple hardware architectures, optimized for resource constrained devices. [`Apache 2.0`](https://directory.fsf.org/wiki/License:Apache2.0)
```
Comment (sjfricke) [inline README.md]: redirects to `http://www.zephyrproject.org/` so just add the `www`

**Comment by sjfricke** [comment]
Hey, Thanks for going through and redirecting to the actual links, I know that takes some time and **really** appreciate it. Please just fix the two comments and I'll merge it in

**Comment by sjfricke** [comment]
So quick last thing, for `kilb` you reference the MIT license from the Atom project, for consistency sake I always just aim to find where the library sources it's license from, the Klib project links `https://en.wikipedia.org/wiki/MIT_License`

**Comment by sjfricke** [comment]
Don't worry, I will squash the commits together so it looks like one commit so these small changes are no issue

**Comment by sambhav2612** [comment]
Sure @sjfricke

**Comment by sambhav2612** [comment]
Can you have a look once again as to why Travis fails for [this link](https://github.com/nasciiboy/recursiveregexpraptor-4) which I can't find myself?

**Comment by sjfricke** [comment]
I added that link in a PR earlier today so you don't see it in your repo.

Thanks again for the great PR

**Pull request merged**